Utils/Dataflow/016_task2es/README

```diff
@@ -9,6 +9,8 @@ Transform task metadata to fit ES scheme:
 * split comma-separated 'hashtag_list' to the actual list structure
 * turn hashtags to lower case
 * split dot-separated 'output_formats' to the actual list structure
+* extract 'chain_id' from 'chain_data' and turn the latter into array
+of integers
```

Comment:
Could you please indent the last line to be aligned with the rest of the text? If later there will be more items, it will make the list look more accurate and prevent stumbling on this line.
Compare:
```
* some item here
* extract 'chain_id' from 'chain_data' and turn the latter into array
  of integers
* yet another item describing the stage
```
vs.
```
* some item here
* extract 'chain_id' from 'chain_data' and turn the latter into array
of integers
* yet another item describing the stage
```

Reply:
Done.